api/frameworks.ts

```diff
@@ -0,0 +1,22 @@
+import { NowRequest, NowResponse } from '@now/node';
+import { withApiHandler } from './_lib/util/with-api-handler';
+import frameworkList, { Framework } from '../packages/frameworks';
+
+const frameworks: Framework[] = (frameworkList as Framework[]).map(
```

Comment:
Hopefully we can avoid `as` in a future PR when `@now/frameworks` is updated 😄

Reply:
Yup :D